sktime/transformations/series/compose.py

```diff
@@ -70,7 +68,7 @@ class OptionalPassthrough(_SeriesToSeriesTransformer):
         "fit-in-transform": True,
     }
 
-    def __init__(self, transformer, passthrough=True):
+    def __init__(self, transformer, passthrough=False):
```

Comment:
Shouldn't this be True by default?

Reply:
so this `OptionalPassthrough` will most likely only be used in combination with a pipeline like `TransformedTargetForecaster` and then a grid search. I actually wanted not set a default value for `passthrough` because user might forget to add this to the grid as `passthrough: [True, False]` which is the only way it makes sense to use this class. However, not giving a default doesnt work either. So I thought to make `passthrough=False` is better than `True` in case user forgets to add it to grid